api/pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.ccci.gto</groupId>
     <artifactId>idm-user-management</artifactId>
-    <version>2.0.5-SNAPSHOT</version>
+    <version>2.0.6</version>
```

Comment:
when we release you should set it to 2.0.5 because that version hasn't been released yet. But I would also want to keep the release update separate from this PR

Reply:
there is also a `mvn release:prepare` and `mvn release:perform` command that will automate all of this

Reply:
Removed version change from this PR.